fix compilation with GCC 11, fixes compilation errors like:
    error: numeric_limits is not a member of std
author: Kenneth Hoste (HPC-UGent)
diff -ru ParaView-v5.9.1.orig/VTK/Common/Core/vtkGenericDataArrayLookupHelper.h ParaView-v5.9.1/VTK/Common/Core/vtkGenericDataArrayLookupHelper.h
--- ParaView-v5.9.1.orig/VTK/Common/Core/vtkGenericDataArrayLookupHelper.h	2021-05-13 18:50:06.000000000 +0200
+++ ParaView-v5.9.1/VTK/Common/Core/vtkGenericDataArrayLookupHelper.h	2021-12-23 11:07:09.825480119 +0100
@@ -25,6 +25,7 @@
 #include "vtkIdList.h"
 #include <algorithm>
 #include <cmath>
+#include <limits>
 #include <unordered_map>
 #include <vector>
 
diff -ru ParaView-v5.9.1.orig/VTK/Common/DataModel/vtkPiecewiseFunction.cxx ParaView-v5.9.1/VTK/Common/DataModel/vtkPiecewiseFunction.cxx
--- ParaView-v5.9.1.orig/VTK/Common/DataModel/vtkPiecewiseFunction.cxx	2021-05-13 18:50:06.000000000 +0200
+++ ParaView-v5.9.1/VTK/Common/DataModel/vtkPiecewiseFunction.cxx	2021-12-23 11:07:41.295765924 +0100
@@ -22,6 +22,7 @@
 #include <cassert>
 #include <cmath>
 #include <iterator>
+#include <limits>
 #include <set>
 #include <vector>
 
diff -ru ParaView-v5.9.1.orig/VTK/Filters/HyperTree/vtkHyperTreeGridThreshold.cxx ParaView-v5.9.1/VTK/Filters/HyperTree/vtkHyperTreeGridThreshold.cxx
--- ParaView-v5.9.1.orig/VTK/Filters/HyperTree/vtkHyperTreeGridThreshold.cxx	2021-05-13 18:50:06.000000000 +0200
+++ ParaView-v5.9.1/VTK/Filters/HyperTree/vtkHyperTreeGridThreshold.cxx	2021-12-23 11:07:51.895861970 +0100
@@ -27,6 +27,7 @@
 #include "vtkHyperTreeGridNonOrientedCursor.h"
 
 #include <cmath>
+#include <limits>
 
 vtkStandardNewMacro(vtkHyperTreeGridThreshold);
 
diff -ru ParaView-v5.9.1.orig/VTK/Rendering/Core/vtkColorTransferFunction.cxx ParaView-v5.9.1/VTK/Rendering/Core/vtkColorTransferFunction.cxx
--- ParaView-v5.9.1.orig/VTK/Rendering/Core/vtkColorTransferFunction.cxx	2021-05-13 18:50:06.000000000 +0200
+++ ParaView-v5.9.1/VTK/Rendering/Core/vtkColorTransferFunction.cxx	2021-12-23 11:07:25.985627071 +0100
@@ -21,6 +21,7 @@
 #include <algorithm>
 #include <cmath>
 #include <iterator>
+#include <limits>
 #include <set>
 #include <vector>
 
